boards/arm/frdm_k64f/Kconfig.defconfig

```diff
@@ -129,7 +132,7 @@ endif # SPI
 if NETWORKING
 
 config NET_L2_ETHERNET
-	def_bool y
+	def_bool y if !RADIO_WCN14A2A
```

Comment:
Maybe this should be !UART_MCUX_2, after all, Ethernet conflicts not with an LTE modem, but with UART2?